### ISSUE TITLE
Reveal.js is not accessible to keyboard and assistive technology users

### ISSUE THREAD

**Issue by marcysutton** [hakimel/reveal.js#932]
There are two core issues:

First: As a screen reader user, I have no way of knowing when the slide changes. Content is not read aloud when hitting the space bar or arrow keys. To fix, focus should be sent to the current slide after firing of the `slidechanged` event so that users know there is new content.

Second: The CSS for Reveal.js leaves all slides in the deck "visible" to screen readers. This causes links in slides besides the current one to be read aloud, providing a terrible experience for users of assistive technology. Similarly, when tabbing through a slide, hidden links from all slides are focusable in the tab order, not just visible links in the current slide.
